packages/sit-onyx/src/components/examples/GridPlayground/GridPlayground.vue

```diff
@@ -1,167 +1,279 @@
+  margin: string;
+  gutter: string;
+  columnCount: number;
+}>();
+
+watch(
+  [viewportSize.width, gridRef, gridSettings],
+  async () => {
+    if (!gridRef.value) return;
+
+    await nextTick();
+    const style = getComputedStyle(gridRef.value);
+
+    gridValues.value = {
+      margin:
+        style.getPropertyValue("--onyx-grid-margin-inline") ||
+        style.getPropertyValue("--onyx-grid-margin"),
+      gutter: style.getPropertyValue("--onyx-grid-gutter"),
+      columnCount: +style.getPropertyValue("--onyx-grid-columns"),
+    };
   },
-} as const;
-export type GridBreakpoints = keyof typeof BREAKPOINTS;
-export type GridElementSettings = { breakpoint?: GridBreakpoints; spans: number }[];
-
-const elements = ref<GridElementSettings[]>([]);
-const selectedElement = ref<GridElementSettings>();
-const gridElement = ref<HTMLElement>();
-const gridSettings = ref<GridSettings>();
-const resizeObserver = new ResizeObserver(() => updateGridSettings());
-
-const isMaxMd = ref(false);
-const isMaxLg = ref(false);
-const isCentered = ref(false);
-const is20Xl = ref(false);
-
-const deleteElement = () => {
-  elements.value = elements.value.filter((e) => e !== selectedElement.value);
-  selectedElement.value = undefined;
+  { immediate: true, deep: true },
+);
+
+const deleteElement = (index: number) => {
+  gridElements.value.splice(index, 1);
+  gridElementIndexToEdit.value = undefined;
 };
 
-const updateGridSettings = () => {
-  const computedGrid = getComputedStyle(gridElement.value!);
-  gridSettings.value = {
-    gridTemplateColumns: computedGrid.gridTemplateColumns,
-    gutterSize: computedGrid.getPropertyValue("--onyx-grid-gutter"),
-    marginSize: computedGrid.getPropertyValue("--onyx-grid-margin"),
-    maxWidth: computedGrid.getPropertyValue("--onyx-grid-max-width"),
-    isCentered: isCentered.value,
-  };
+const updateElement = (index: number, newElement: GridElementConfig) => {
+  gridElements.value[index] = newElement;
+  gridElementIndexToEdit.value = undefined;
 };
 
-watch([isMaxMd, isMaxLg, isCentered, is20Xl], () => updateGridSettings());
+const alignmentOptions: SelectOption[] = [
+  { label: "left", value: "left" },
+  { label: "center", value: "center" },
+];
 
-onMounted(() => {
-  resizeObserver.observe(document.body);
-  resizeObserver.observe(gridElement.value!);
-});
+const maxWidthOptions: SelectOption[] = [
+  { label: "none", value: "none" },
+  { label: `${ONYX_BREAKPOINTS.lg}px`, value: "md" },
+  { label: `${ONYX_BREAKPOINTS.xl}px`, value: "lg" },
+];
 
-onBeforeUnmount(() => {
-  resizeObserver.unobserve(document.body);
-  resizeObserver.unobserve(gridElement.value!);
-});
+const maxColumnsOptions: SelectOption[] = [
+  { label: "16", value: 16 },
+  { label: "20", value: 20 },
+];
 
-const handleDeleteModifier = () => {
-  if (!selectedElement.value) return;
-  selectedElement.value.splice(elements.value.indexOf(selectedElement.value), 1);
-};
+const currentBreakpoint = computed(() => {
```

Comment:
in the column config, the breakpoints are uppercase (e.g. 2XS), here they're lowercase (2xs)
-> could you align them for a more consistent experience?